test/e2e/framework/util.go

```diff
@@ -2317,36 +2317,16 @@ func retryCmd(command string, args ...string) (string, string, error) {
 	return stdout, stderr, err
 }
 
-// WaitForStableCluster waits until all existing pods are scheduled and returns their amount.
-func WaitForStableCluster(c clientset.Interface, masterNodes sets.String) int {
-	timeout := 10 * time.Minute
-	startTime := time.Now()
-
-	allPods, err := c.CoreV1().Pods(metav1.NamespaceAll).List(metav1.ListOptions{})
-	ExpectNoError(err)
-	// API server returns also Pods that succeeded. We need to filter them out.
-	currentPods := make([]v1.Pod, 0, len(allPods.Items))
-	for _, pod := range allPods.Items {
-		if pod.Status.Phase != v1.PodSucceeded && pod.Status.Phase != v1.PodFailed {
-			currentPods = append(currentPods, pod)
-		}
-
+// ListNamespaceEvents lists the events in the given namespace.
+func ListNamespaceEvents(c clientset.Interface, ns string) error {
```

Comment:
there is already DumpEventsInNamespace.
you can either:
- use DumpEventsInNamespace with it's sorting option
- add optional sorting to DumpEventsInNamespac as a boolean flag or a sort.Interface:
https://golang.org/pkg/sort/#Interface

Reply:
This diff is an error in rebasing; I did not add this function... fixing...